src/fmt.rs

```diff
@@ -0,0 +1,74 @@
+use serde::{Deserialize, Serialize};
```

Comment:
I think let's use `formats` for this file, rather than "semantically conflicting" with `std::fmt`. Also the plural makes it more consistent with neighbouring mods

Reply:
Done!